.github/workflows/python-package.yml

```
@@ -5,35 +5,34 @@ name: Python package
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7]
+        python-version: [3.6, 3.7, 3.8]
```

A lot of distributions are starting to build against Python 3.10. I think that at least Python 3.9 should be added.

```suggestion
        python-version: [3.6, 3.7, 3.8, 3.9]
```

---

That's worth a try of course